### PR TITLE
Remove service details content type.

### PR DESCRIPTION
**Pull request by weitzman**
**Jira:** (Skip unless you are MA staff)
DP-29606

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by tom-fleming** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 37cf0895e9999d23166d605da74f9d6dc8724d54
</p>

File: conf/drupal/config/metatag.metatag_defaults.node__campaign_landing.yml [hunk 1]
```diff
@@ -11,6 +11,4 @@ tags:
   twitter_cards_image: '[node:field_header:entity:field_image:social_media]'
   twitter_cards_description: '[node:field_meta_description]'
   twitter_cards_type: summary_large_image
-  mass_metatag_organization: '[node:mass_organizations]'
```
Comment (arthurbaghdas) [inline conf/drupal/config/metatag.metatag_defaults.node__campaign_landing.yml]: This is not related to service details and should not be removed.

**Review by arthurbaghdas — CHANGES_REQUESTED** [review]
@weitzman is there a reason we want to keep the mass_migrate module in place?

File: conf/drupal/config/metatag.metatag_defaults.node__campaign_landing.yml (hunk 1 above)
Comment (weitzman) [reply conf/drupal/config/metatag.metatag_defaults.node__campaign_landing.yml]: Good catch. Reverted.

**Comment by weitzman** [comment]
> @weitzman is there a reason we want to keep the mass_migrate module in place?

Good catch. I have uninstalled the module. Lets remove the code it in a subsequent PR.

**Review by arthurbaghdas — APPROVED** [review]
